src/overlays/actors/ovl_En_Door/z_en_door.h

```diff
@@ -22,11 +22,14 @@
  *
  */
 
-typedef struct {
-    /* 0x00 */ s16 sceneNum;
-    /* 0x02 */ u8 dListIndex;
-    /* 0x04 */ s16 objectId;
-} EnDoorInfo;
+#define ENDOOR_PARAMS_TYPE_SHIFT 7
+#define ENDOOR_PARAMS_TYPE_MASK (7 << ENDOOR_PARAMS_TYPE_SHIFT)
+#define ENDOOR_TYPE(thisx) ((thisx)->params >> ENDOOR_PARAMS_TYPE_SHIFT & 7)
```

Comment:
I think these macros should also be named as getters (ie. they should include GET, or either IS or CHECK for booleans)